recipes/python-statuspage/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set url_version = "0.3-pre" %}
+{% set version = url_version.replace("-", "") %}
+
+package:
+  name: python-statuspage
```

Comment:
I dropped the prefix as they have now added a package to PyPI under this name. Not sure how we feel about this. Though it seems like the current trend is away from prefixing.